src/app/plugins-initializer.ts

```diff
@@ -27,8 +29,7 @@ const loadSinglePlugin = async (pluginDefinition: PluginDefinition): Promise<str
     const scriptsLoad = pluginDefinition.scripts.map((script) => loadScript(script));
     await Promise.all(scriptsLoad);
   } catch (e) {
-    console.log(`Module(s) ${pluginDefinition.angularModules.join(', ')} load fail`);
-    console.log(e);
+    console.log(`Module(s) ${pluginDefinition.angularModules.join(', ')} load fail`, e);
```

Comment:
We should use console.error here, because this is an error that should never occure for users and if it does, needs to be easy to detect.